### DISCUSSION TITLE
Where do I find the new ViT baseline param?

### DISCUSSION THREAD

**Discussion by sayakpaul** [google-research/big_vision#73] [Q&A] [answered]
Regarding https://github.com/google-research/big_vision#vit-baseline

**Comment by xiaohuazhai** [comment]
It's available here: 

https://github.com/google-research/big_vision/blob/main/big_vision/configs/vit_s16_i1k.py

You can always find all our configs under `configs` folder.

**Comment by sayakpaul** [comment]
Sorry, I should have been clearer. By param, I meant the pre-trained params. Are they available yet?

**Comment by akolesnikoff** [comment]
It is an oversight from our side: we will publish it soon. Leaving the issue open until then.

**Comment by sayakpaul** [comment]
Thanks! 

As soon as they're available I will try to port them to a TensorFlow / Keras friendly model for wider community adoption.

**Comment by akolesnikoff** [comment, marked as answer]
Models have been landed, see https://github.com/google-research/big_vision/blob/6c376d6f621fe93627ee697674abd202392d8faa/big_vision/models/vit.py#L328

Closing the issue, feel free to reopen if you have any questions.
